### PR TITLE
Update protect-documents-that-have-fci-or-other-properties.md

### PR DESCRIPTION
**Pull request by aamalhot**
Update for Bug 1956254

### CODE REVIEW

**Comment by aamalhot** [comment]
@chrfox please review

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/protect-documents-that-have-fci-or-other-properties.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ Before you can use a Windows Server FCI property or other property in a DLP poli
 
 In SharePoint Online and OneDrive for Business, the search index is built up by crawling the content on your sites. The crawler picks up content and metadata from the documents in the form of crawled properties. The search schema helps the crawler decide what content and metadata to pick up. Examples of metadata are the author and the title of a document. However, to get the content and metadata from the documents into the search index, the crawled properties must be mapped to managed properties. Only managed properties are kept in the index. For example, a crawled property related to author is mapped to a managed property related to author.
 
+> [!NOTE]
+> Make sure to use managed property name and not crawled property name when creating DLP rules using `ContentPropertyContainsWords` condition.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/protect-documents-that-have-fci-or-other-properties.md]:
```suggestion
> Make sure to use a managed property name and not a crawled property name when creating DLP rules using the `ContentPropertyContainsWords` condition.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by chrfox — COMMENTED** [review]
You have a typo in the first word of your note.  It reads "Bue" when I think it should be "Be"

**Pull request merged**